src/components/playButton/PlayButton.styles.tsx

```diff
@@ -1,17 +1,13 @@
 import styled from 'styled-components'
+import { compose, border, position, space } from 'styled-system'
 import { colors } from '../../theme/colors'
-import { mediaQueries } from '../../theme/mediaQueries'
-import { space } from '../../theme/space'
+import { PlayButtonProps } from './PlayButton.types'
 
-export const ButtonWrapper = styled.button`
-  padding: ${space.lg};
+export const Button = styled.button<PlayButtonProps>`
+  /* can't use styled-system here due to types issue and ColorProps properties */
```

Comment:
Omit color from React.ButtonHTMLAttributes<HTMLButtonElement> in the PlayButtonProps so the type doesnt conflict

Reply:
I did try this approach but received a type error still. I've updated the branch to show the issue more clearly 😄

Reply:
Moved to a super verbose `styled(props => <button {...props} />)` approach. Known issue with `styed-components` v < 5. We can resolve this once we move to v5.